src/main/java/com/activestate/cloudfoundryjenkins/DeploymentInfo.java

```diff
@@ -39,6 +39,9 @@
     private String buildpack;
     private String command;
     private String domain;
+    private  boolean  isNewAppToBeCreated  = false;
```

Comment:
rename this to createNewApp , do the rename of  corresponding getter/setter accordingly